pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -694,26 +694,33 @@ func TestRestartableInitContainers(t *testing.T) {
 		name                    string
 		pod                     *v1.Pod
 		enableSidecarContainers bool
-		wantStatus              *framework.Status
+		wantPreFilterStatus     *framework.Status
+		wantFilterStatus        *framework.Status
 	}{
 		{
-			name: "allow pod without restartable init containers if sidecar containers is disabled",
-			pod:  newPod(),
+			name:                "allow pod without restartable init containers if sidecar containers is disabled",
+			pod:                 newPod(),
+			wantPreFilterStatus: framework.NewStatus(framework.Success),
+			wantFilterStatus:    framework.NewStatus(framework.Success),
 		},
 		{
-			name:       "not allow pod with restartable init containers if sidecar containers is disabled",
-			pod:        newPodWithRestartableInitContainers(),
-			wantStatus: framework.NewStatus(framework.UnschedulableAndUnresolvable, "Pod has a restartable init container and the SidecarContainers feature is disabled"),
+			name:                "not allow pod with restartable init containers if sidecar containers is disabled",
+			pod:                 newPodWithRestartableInitContainers(),
+			wantPreFilterStatus: framework.NewStatus(framework.UnschedulableAndUnresolvable, "Pod has a restartable init container and the SidecarContainers feature is disabled"),
 		},
 		{
 			name:                    "allow pod without restartable init containers if sidecar containers is enabled",
 			enableSidecarContainers: true,
 			pod:                     newPod(),
+			wantPreFilterStatus:     framework.NewStatus(framework.Success),
```

Comment:
Do we need to explicitly specify `framework.NewStatus(framework.Success)`?
`framework.NewStatus(framework.Success)` equals `nil`.

Reply:
Do you mean we can set `wantPreFilterStatus = nil` ?

Reply:
Yes, we consider nil as Success.
```
// Code returns code of the Status.
func (s *Status) Code() Code {
	if s == nil {
		return Success
	}
	return s.code
}

```

Reply:
done